src/lib/vm/vm.js

```diff
@@ -1175,6 +1176,13 @@ class VmStack {
       ) {
         const keyword = code.object.name;
         if (keyword in Keywords) {
+          // Special case for Promise.all and Promise.any
+          if (keyword === 'Promise' && ['all', 'any'].includes(code.property.name)) {
+            return {
+              obj: Promise,  // Return the Promise object itself
+              key: code.property.name
+            };
+           }
```

Comment:
Why is this special case needed? Wouldn't it work without it similar to `Date` object?

Reply:
Got it. Looks fine